meson.build

```diff
@@ -207,17 +207,17 @@ elif get_option('bundle') and host_machine.system() == 'darwin'
     )
 else
     lite_bindir = 'bin'
-    lite_docdir = 'share/doc/lite-xl'
-    lite_datadir = 'share/lite-xl'
+    lite_docdir = get_option('datadir') + '/doc/lite-xl'
```

Comment:
In lieu of #1938, we should adopt `/` instead of `+`?

Reply:
I'll rebase when either PR is merged and update the path segment accordingly